src/StakingManager.sol

```diff
@@ -395,6 +340,10 @@ contract StakingManager is
         bytes32 depositDataRoot = depositRootGenerator.generateDepositRoot(_depositData.publicKey, _depositData.signature, withdrawalCredentials, _depositAmount);
         require(depositDataRoot == _depositData.depositDataRoot, "WRONG_ROOT");
 
+        bytes32 fullHash = keccak256(abi.encode(_validatorId, _staker, _tNftRecipient, _bNftRecipient));
+        bytes10 truncatedHash = bytes10(fullHash);
```

Comment:
by birthday paradox approximation, collisions become likely once we hit 1 trillion validators :)

Reply:
hope that happens!